zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -680,6 +680,52 @@ paths:
           description: >
             The Decision Requirements Search Query failed.
             More details are provided in the response body.
+  /authorizations:
+    post:
+      tags:
+        - Authorization
+      summary: Create an authorization
+      operationId: createAuthorization
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/AuthorizationAssignRequest"
+        required: true
+      responses:
+        "204":
+          description: |
+            The Authorization was created successfully.
```

Comment:
❓ There is no content in this Response. Don't you need the key Zeebe generated to update/delete the authorization later?

Reply:
The resource key will come back with the search responses where the key would come from for updates/deletes. Should we also return the record we store here too?

Reply:
No, if you don't need the key directly to store it somewhere Identity-side and it's good enough to read it from the exported records that's fine by me. Thanks!